backend/tournesol/resources/export_readme.txt

```diff
@@ -18,43 +18,116 @@ List of files
 comparisons.csv
 ---------------
 
-Type: algorithm input.
-
 This file contains all the public comparisons made by users. A comparison is
 represented by a rating given by a user for a specific criteria and a pair of
 videos.
 
+List of columns:
+
+- public_username:
+  The username of the user who submitted the comparison.
+- video_a:
+  One of the two video being compared. If the score is negative, it indicates
+  that the user prefers video_a against video_b.
+- video_b:
+  The second video being compared. If the score is positive, it indicates that
+  the user prefers video_b against video_a.
+- criteria:
+  The name of the criteria for the comparison. The main criteria is
```

Comment:
Sometimes we use criterion for the singular and sometimes we use criteria.

To be more clear we should use the same word to define the same concept in the documentation. How about using criterion only for the singular and criteria for the plural?

Reply:
Use "criterion" for the singlular